### ISSUE TITLE
When editing table and on selecting it goes to blank white screen

### ISSUE THREAD

**Issue by RjPatil27** [slashbaseide/slashbase#126]

**Comment by paraswaykole** [comment]
It happened in the desktop IDE when connected to a MySQL or PostgreSQL database.

**Comment by ahmadniaz** [comment]
@paraswaykole Don't mind the question, please. It's my first-time contributing to an open source project. I am a Javascript developer. Is this project for me? As it has the tFrontend label.

**Comment by paraswaykole** [comment]
Yes! This issue is related to javascript and frontend specific. You'd need Golang and Wails installed. Please read and follow the contributing.md file. If you have additional queries, join in our discord server and we will help you setup the project.
